config/prod.exs

```diff
@@ -12,9 +12,10 @@ import Config
 config :screenplay, ScreenplayWeb.Endpoint,
   cache_static_manifest: "priv/static/cache_manifest.json",
   http: [:inet6, port: 4000],
-  redirect_http?: true,
   server: true,
-  url: [port: 80]
+  url: [port: 443],
```

Comment:
It feels so odd that we (as in, computers) use different ports to indicate HTTP vs. HTTPS. I guess I'm still a little fuzzy on port mechanics, but after reading more on this it seems to make sense.